### PR TITLE
Inherit L2GovernorMetadata

### PR DESCRIPTION
**Pull request by alexkeating**
#### Description

- Modify contracts to work with Tally end to end

### CODE REVIEW

**Comment by alexkeating** [comment]
Current dependencies on/for this PR:
* main
  * **PR #85** <a href="https://app.graphite.dev/github/pr/ScopeLift/l2-flexible-voting/85" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/ScopeLift/l2-flexible-voting/85?utm_source=stack-comment).

File: src/L2GovernorMetadata.sol [hunk 1]
```diff
@@ -51,8 +51,8 @@ abstract contract L2GovernorMetadata {
         new uint256[](0),
         new string[](0),
         new bytes[](0),
-        voteStart,
-        voteEnd,
+        block.number,
+        block.number + 43_200,
```
Comment (alexkeating) [inline src/L2GovernorMetadata.sol:55]: We need to make an issue to calculate this correctly

Comment (alexkeating) [reply src/L2GovernorMetadata.sol:55]: https://github.com/ScopeLift/l2-flexible-voting/issues/86

**Review by apbendi — APPROVED** [review]
Looks good to me @alexkeating, nice work.

**Pull request merged**